assets/dev/js/editor/elements/elements/base.js

```diff
@@ -0,0 +1,36 @@
+export default class ElementBase {
```

Comment:
Maybe rename the folder to `elements/types`

Reply:
Handled.